app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -3201,28 +3213,31 @@ class BrowserTabFragment :
                     }
                 },
             )
-            webViewClient.configureWebView(
-                it,
-                object : WebViewCompatMessageCallback {
-                    override fun process(
-                        context: String,
-                        featureName: String,
-                        method: String,
-                        id: String?,
-                        data: JSONObject?,
-                        onResponse: (JSONObject) -> Unit,
-                    ) {
-                        viewModel.webViewCompatProcessJsCallbackMessage(
-                            context = context,
-                            featureName = featureName,
-                            method = method,
-                            id = id,
-                            data = data,
-                            onResponse = onResponse,
-                        )
-                    }
-                },
-            )
+            lifecycleScope.launch {
```

Comment:
On the next PR I'm gonna simplify this and launch the coroutine from the ViewModel instead